docs/topics/hacking/debugging.rst

```diff
@@ -0,0 +1,47 @@
+=========
+Debugging
+=========
+
+The :ref:`docker setup <install-with-docker>` uses supervisord to run the
+django runserver. This means if you want to access the management server
+from a shell to run things like ipdb_ you still can.
```

Comment:
Ah right, I forgot about that. Thanks.

On Fri, Sep 18, 2015 at 10:01 AM, Stuart Colville <notifications@github.com>
wrote:

> In docs/topics/hacking/debugging.rst
> <https://github.com/mozilla/olympia/pull/762#discussion_r39877162>:
>
> > @@ -0,0 +1,47 @@
> > +=========
> > +Debugging
> > +=========
> > +
> > +The :ref:`docker setup <install-with-docker>` uses supervisord to run the
> > +django runserver. This means if you want to access the management server
> > +from a shell to run things like ipdb_ you still can.
>
> Ah here it is:
> https://github.com/mozilla/marketplace-env/blob/master/docker/base-images/python27/Dockerfile#L14
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/olympia/pull/762/files#r39877162>.
>